pkg/jaeger/store/trace_query.go

```diff
@@ -74,7 +85,12 @@ const (
 	  s_url.url,
 	  inst_lib.name,
 	  inst_lib.version,
-	  inst_lib_url.url`
+	  inst_lib_url.url,
+		links_linked_trace_ids,
+		links_linked_span_ids,
+		links_trace_states,
+		links_dropped_tags_count,
+		links_tags`
```

Comment:
Using a whole-row reference looks better and probably performs better too
```suggestion
		link`
```